examples/09_cricket_umpire/run.sh

```
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+echo "🏏 Starting Cricket DRS AI System"
+echo "=================================="
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+cd "$SCRIPT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)